From 214efe6580472336538d74132977e21b2ae483ff Mon Sep 17 00:00:00 2001
From: kholoud mohamed <kholoudm@google.com>
Date: Thu, 20 May 2021 13:50:00 +0100
Subject: [PATCH] Grant MANAGE_APP_OPS_MODES to Managed Provisioning

This is needed to reset INTERACT_ACROSS_PROFILES appop during OTA
update.

Bug: 188665727
Bug: 195630721
Test: N/A
Change-Id: Ibc47abfb32ad2e0cb68ebe9290f99ab3082d9ebd
Merged-In: Ibc47abfb32ad2e0cb68ebe9290f99ab3082d9ebd
(cherry picked from commit 92105a2ed570f1e829024d8fec0374156ba2cff5)
(cherry picked from commit 0dd1b6508bb4a6ba350fedb8ea95df9ab4ed3d9c)
---
 AndroidManifest.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 4de3b714..2462d5b0 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -34,6 +34,7 @@
     <uses-permission android:name="android.permission.BIND_DEVICE_ADMIN" />
     <uses-permission android:name="android.permission.MANAGE_DEVICE_ADMINS" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
+    <uses-permission android:name="android.permission.MANAGE_APP_OPS_MODES"/>
 
     <uses-permission android:name="android.permission.CHANGE_COMPONENT_ENABLED_STATE" />
     <uses-permission android:name="android.permission.CHANGE_CONFIGURATION" />
-- 
2.17.1

